views/play.erb

```diff
@@ -0,0 +1,10 @@
+<h1> Welcome <%= @name %> </h1>
```

Comment:
Hosted the game on my computer and everything worked (filling in name, clicking buttons)